apps/web/src/services/cost-explorer/lib/config.ts

```diff
@@ -23,13 +23,6 @@ export const MORE_GROUP_BY = {
     ADDITIONAL_INFO: 'additional_info',
 };
 
-export const GRANULARITY_ITEM_MAP = {
-    [GRANULARITY.ACCUMULATED]: { name: GRANULARITY.ACCUMULATED, label: 'Accumulated' },
-    [GRANULARITY.DAILY]: { name: GRANULARITY.DAILY, label: 'Daily' },
-    [GRANULARITY.MONTHLY]: { name: GRANULARITY.MONTHLY, label: 'Monthly' },
-    [GRANULARITY.YEARLY]: { name: GRANULARITY.YEARLY, label: 'Yearly' },
-} as const;
```

Comment:
question: Is it a variable that has never been used before?

Reply:
Yes. I'm not sure why I made this😂